### PR TITLE
Include global date filter to Drill to URL post message

### PR DESCRIPTION
**Pull request by qduval**
---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/770/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2357/ : FAILURE in 2m 28s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/771/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2358/ : SUCCESS in 11m 30s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/783/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2372/ : SUCCESS in 12m 07s

File: libs/sdk-embedding/src/iframe/kd.ts [hunk 1]
```diff
@@ -931,11 +931,19 @@ export namespace EmbeddedKpiDashboard {
...
+    export type DrillToUrlFilters = Array<
+        EmbeddedGdc.DashboardDateFilter | EmbeddedGdc.IDashboardAttributeFilter
+    >;
+
     /**
      * @public
      */
     export interface IDrillToUrlStartedDataBody {
         id: string;
+        filters: DrillToUrlFilters;
```
Comment (lupko) [inline libs/sdk-embedding/src/iframe/kd.ts:946]: please add few lines of docs for this new prop. write what are the values of these filters (e.g. values of dashboard filters set when user triggered the drill).

also give hint on the existence of the type-guards. "You can use the typeguards defined in EmbeddedGdc namespace to test the type of the filter. "

File: libs/sdk-embedding/src/iframe/kd.ts [hunk 2]
```diff
@@ -944,6 +952,7 @@ export namespace EmbeddedKpiDashboard {
     export interface IDrillToUrlResolvedDataBody {
         id: string;
         url: string;
+        filters: DrillToUrlFilters;
```
Comment (lupko) [inline libs/sdk-embedding/src/iframe/kd.ts:955]: same doc here

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/793/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2380/ : SUCCESS in 24m 01s

**Review by lupko — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/334/ : SUCCESS in 29s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/590/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2382/ : SUCCESS in 17m 56s

**Pull request merged**